files/image_config/warmboot-finalizer/finalize-warmboot.sh

```diff
@@ -52,6 +52,16 @@ function check_warm_boot()
     WARM_BOOT=`sonic-db-cli STATE_DB hget "WARM_RESTART_ENABLE_TABLE|system" enable`
 }
 
+function check_fast_reboot()
+{
+    SYSTEM_FAST_REBOOT=`sonic-db-cli STATE_DB GET "FAST_REBOOT|system"`
```

Comment:
It might help to put a debug print here to announce if fast-reboot is enabled/disable. I think we should do same for warmreboot.

Reply:
As I wrote in the other comment in case warm-reboot is disabled it is being printed to log and script exits. In that case is it required to add also print for the case it is enabled? It will print when warm-reboot is finalized.

Let me know what you think. In the meantime I will add debug prints for fast-reboot check.

Reply:
No change needed in that case. Thanks!